incubator/kubevirt/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+name: kubevirt
+home: https://github.com/kubevirt/kubevirt
+version: 0.1.0
```

Comment:
Here 0.1.0 is given, but the images are pulled from `latest` by default, this is inconsistent.

I'd suggest to update this to 0.2.0 (which latest is pointing to).

Reply:
It chart version, it a little strange that I have version `0.2.0` without `0.1.0`, but it can be a good idea to couple it with KubeVirt releases